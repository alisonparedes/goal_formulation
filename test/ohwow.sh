python ../src/ohwow.py ../test/initial_state_01.txt 0 3
python ../src/ohwow.py ../test/initial_state_02.txt 0 3
python ../src/ohwow.py ../test/initial_state_03.txt 0 3
python ../src/ohwow.py ../test/initial_state_04.txt 0 3
python ../src/ohwow.py ../test/initial_state_05.txt 0 3
python ../src/ohwow.py ../test/initial_state_06.txt 0 3
python ../src/ohwow.py ../test/initial_state_07.txt 0 3
python ../src/ohwow.py ../test/initial_state_08.txt 0 3
python ../src/ohwow.py ../test/initial_state_09.txt 0 3
python ../src/ohwow.py ../test/initial_state_10.txt 1 3
python ../src/ohwow.py ../test/initial_state_11.txt 1 3
python ../src/ohwow.py ../test/initial_state_12.txt 1 3
python ../src/ohwow.py ../test/initial_state_13.txt 1 3
python ../src/ohwow.py ../test/initial_state_14.txt 1 3
python ../src/ohwow.py ../test/initial_state_15.txt 1 3
python ../src/ohwow.py ../test/initial_state_16.txt 1 3
# more food
python ../src/ohwow.py ../test/initial_state_17.txt 4 10
python ../src/ohwow.py ../test/initial_state_18.txt 4 10


