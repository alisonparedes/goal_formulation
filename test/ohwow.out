initial_state: ../test/initial_state_01.txt
max_food: 0
number_of_samples: 3
[[3, 0], [0, 0]]
F- F?
-b ?b

F- F?
-b ?b

F- F?
-b ?b

initial_state: ../test/initial_state_02.txt
max_food: 0
number_of_samples: 3
[[3, 0], [0, 0]]
F- F?
HB HB

F- F?
HB HB

F- F?
HB HB

initial_state: ../test/initial_state_03.txt
max_food: 0
number_of_samples: 3
[[0, 0], [0, 0]]
H- H?
-B ?B

H- H?
-B ?B

H- H?
-B ?B

initial_state: ../test/initial_state_04.txt
max_food: 0
number_of_samples: 3
[[0, 0], [0, 0]]
-- ??
HB HB

-- ??
HB HB

-- ??
HB HB

initial_state: ../test/initial_state_05.txt
max_food: 0
number_of_samples: 3
[[0, 0], [0, 0]]
-$ ?$
-B ?B

-$ ?$
-B ?B

-$ ?$
-B ?B

initial_state: ../test/initial_state_06.txt
max_food: 0
number_of_samples: 3
[[0, 0], [0, 0]]
$- $?
-B ?B

$- $?
-B ?B

$- $?
-B ?B

initial_state: ../test/initial_state_07.txt
max_food: 0
number_of_samples: 3
[[3, 0], [0, 0]]
F- F?
HB HB

F- F?
HB HB

F- F?
HB HB

initial_state: ../test/initial_state_08.txt
max_food: 0
number_of_samples: 3
[[0, 0], [0, 0]]
-- ??
-* ?*

-- ??
-* ?*

-- ??
-* ?*

initial_state: ../test/initial_state_09.txt
max_food: 0
number_of_samples: 3
[[3, 0], [0, 0]]
F- F?
-b ?b

F- F?
-b ?b

F- F?
-b ?b

initial_state: ../test/initial_state_10.txt
max_food: 1
number_of_samples: 3
[[3, 0], [0, 0]]
FH FH
-B ?B

FH FH
-B ?B

FH FH
-B ?B

initial_state: ../test/initial_state_11.txt
max_food: 1
number_of_samples: 3
[[3, 0], [0, 0]]
F- F?
HB HB

F- F?
HB HB

F- F?
HB HB

initial_state: ../test/initial_state_12.txt
max_food: 1
number_of_samples: 3
[[0, 1], [2, 0]]
$F $?
-B ?B

$- $?
FB ?B

$F $?
-B ?B

initial_state: ../test/initial_state_13.txt
max_food: 1
number_of_samples: 3
[[3, 0], [0, 0]]
F# F#
-b ?b

F# F#
-b ?b

F# F#
-b ?b

initial_state: ../test/initial_state_14.txt
max_food: 1
number_of_samples: 3
[[3, 0], [0, 0]]
F# F#
HB HB

F# F#
HB HB

F# F#
HB HB

initial_state: ../test/initial_state_15.txt
max_food: 1
number_of_samples: 3
[[0, 3], [0, 0]]
$# $#
FB ?B

$# $#
FB ?B

$# $#
FB ?B

initial_state: ../test/initial_state_16.txt
max_food: 1
number_of_samples: 3
[[3, 0], [0, 0]]
F# F#
-* ?*

F# F#
-* ?*

F# F#
-* ?*

initial_state: ../test/initial_state_17.txt
max_food: 4
number_of_samples: 10
[[10, 3, 2, 1], [0, 0, 4, 0], [2, 3, 2, 2], [5, 3, 3, 0]]
F#-F F#??
F#-F ?#??
---- ????
-#-b ?#?b

F#F- F#??
-#F- ?#??
F--- ????
-#-b ?#?b

F#-F F#??
-#-- ?#??
-F-F ????
-#-b ?#?b

F#-- F#??
-#-F ?#??
-F-- ????
F#-b ?#?b

F#-- F#??
-#-- ?#??
-FF- ????
-#Fb ?#?b

F#-F F#??
-#F- ?#??
---F ????
-#-b ?#?b

F#-F F#??
-#-- ?#??
-F-- ????
-#Fb ?#?b

F#FF F#??
F#-- ?#??
---- ????
-#-b ?#?b

F#-- F#??
-#FF ?#??
--F- ????
-#-b ?#?b

F#-- F#??
F#-- ?#??
F--F ????
-#-b ?#?b

initial_state: ../test/initial_state_18.txt
max_food: 4
number_of_samples: 10
[[10, 3, 3, 2], [0, 0, 6, 0], [0, 2, 1, 3], [2, 5, 3, 0]]
F#HF F#H?
F#-F ?#??
---- ????
-#-B ?#?B

F#H- F#H?
-#F- ?#??
FF-- ????
-#-B ?#?B

F#HF F#H?
-#-- ?#??
FF-- ????
-#-B ?#?B

F#H- F#H?
-#-F ?#??
-F-- ????
F#-B ?#?B

F#H- F#H?
-#-F ?#??
--FF ????
-#-B ?#?B

F#H- F#H?
-#-F ?#??
-F-- ????
-#FB ?#?B

F#H- F#H?
F#-- ?#??
---- ????
F#FB ?#?B

F#H- F#H?
F#F- ?#??
---F ????
-#-B ?#?B

F#H- F#H?
-#-F ?#??
FF-- ????
-#-B ?#?B

F#H- F#H?
-#-- ?#??
-F-F ????
-#FB ?#?B

